ft_prepare_mesh.m

```diff
@@ -22,12 +22,24 @@
 %   cfg.downsample  = integer number (default = 1, i.e. no downsampling), see FT_VOLUMEDOWNSAMPLE
 %   cfg.spmversion  = string, 'spm2', 'spm8', 'spm12' (default = 'spm8')
 %
-% For method 'headshape you should specify
+% For method 'headshape' you should specify
 %   cfg.headshape   = a filename containing headshape, a Nx3 matrix with surface
 %                     points, or a structure with a single or multiple boundaries
 %
 % For method 'cortexhull' you should not give input data, but specify
-%   cfg.headshape   = sting, filename containing the pial surface computed by freesurfer recon-all
+%   cfg.headshape   = string, filename containing the pial surface computed by freesurfer recon-all
+%
+% For method 'fittemplate' you should specify
+%   cfg.headshape   = a filename containing headshape
```

Comment:
it does not explain which one is fitted to which

Reply:
I was just wondering where to explain this. Would it make sense to put the description of the method to line 35 (so replacing the example)?

Reply:
@robertoostenveld what do you think about this suggestion?

Reply:
an example is not explanation. Just add a sentence like 

"with this method you are fitting the headshape from the configuration to the template; the resulting affine transformation is applied to the input mesh (or set of meshes), which is subsequently returned as output variable.

Reply:
Added this documentation